jet_tracking/devices.py

```diff
@@ -866,3 +910,24 @@ class Diffract(Device, _TableMixin):
                         doc='Diffraction periodogram Frequency analysis amplitude array')
     state = Cpt(EpicsSignal, ':STATE',
                 doc='State of diffraction analysis')
+
+
+# classes used for jet tracking testing
+class JTInput(Device):
+    nframe = Cpt(EpicsSignal, ':NFRAME', doc='number of frames passed')
+    i0 = Cpt(EpicsSignal, ':i0', doc='Wave8')
+    evtcode = Cpt(EpicsSignal, ':EVTCODE', doc='event code')
+    mtr = Cpt(EpicsSignal, ':MTR', doc='motor position')
+    mtr_prec = Cpt(EpicsSignal, ':MTR_PREC', doc='motor precision')
+
+
+class JTOutput(Device):
+    nframe = Cpt(EpicsSignal, ':NFRAME', doc='number of frames used')
+    det = Cpt(EpicsSignal, ':DET', doc='detector intensity')
+    i0 = Cpt(EpicsSignal, ':I0', doc='Wave8')
+    mtr = Cpt(EpicsSignal, ':MTR', doc='motor position')
+
+
+class JTFake(Device):
+    stopper = Cpt(EpicsSignal, ':STOPPER', doc='fake stopper')
```

Comment:
Are these fake "states"?

Reply:
I feel that we should maybe move the "fake" or "test" stuff outside in its own file. I'm not sure if this is only temporary or if we want to keep a suite of code&PVs for testing around, but I don't think it belongs in the main code.